src/attribute.rs

```diff
@@ -0,0 +1,899 @@
+use std::str::FromStr;
+
+use anyhow::{anyhow, Result};
+use strum_macros::{Display, EnumIter, EnumString};
+
+#[derive(Debug, EnumString, PartialEq, EnumIter, Display)]
+pub enum RawEventType {
+    #[strum(serialize = "bootp")]
```

Comment:
First, I was wondering in which contexts this strum representation is needed. Second, since all are now in lowercase, we can simply place this line at the top:

```rust
#[strum(serialize_all = "lowercase")]
pub enum RawEventType {
```

Reply:
However, if this strum representation is only for users, I think the following would better:

Connection, DHCP, Kerberos, Log, NFS, and so on.

Reply:
1. The RawEventType enum was added to provide a list of types of raw events that the user can choose from in the part of the UI that creates the triage policy.
```rust
   // ["bootp", "conn", "dhcp", "dns", "ftp", "http", "kerberos", "ldap", "log", "mqtt", 
   //   "network", "nfs", "ntlm", "rdp", "smb", "smtp", "ssh", "tls", "window"]
   let enum_strings: Vec<String> = RawEventType::iter().map(|attr| attr.to_string()).collect();
```
2. Where can I refer to for how to represent the raw event type in the UI?